pkg/urbit/vere/io/http.c

```diff
@@ -1857,11 +2122,34 @@ _http_io_exit(u3_auto* car_u)
   //   _http_serv_close(htp_u);
   // }
 
-  // XX close u3_Host.fig_u.cli_u and con_u
+  {
+    u3_noun dat = u3nt(u3_nul, 25, u3i_string("data:urbit shutting down\n\n"));
```

Comment:
I think that's a little long to hardcode the count. I think it'd be better to measure explicitly here.